changelog.adoc

```diff
@@ -6,8 +6,8 @@
 
 Bugfixes
 
-* Fixes a bug which would not allow users to utilize "--anonymous" or
-  "--all-authenticated-users" when creating an endpoint permission.
+* Fixes a bug which would not allow users to utilize ``--anonymous`` or
+  ``--all-authenticated`` when creating an endpoint permission.
```

Comment:
```suggestion
* Fixes a bug which would not allow users to utilize `--anonymous` or
  `--all-authenticated` when creating an endpoint permission.
```

I _believe_ that single backtick and double backtick are both accepted by asciidoc/docbook, but I'd rather play it safe. As far as I know, ReST is the only markdown language with the double-backtick syntax.

Reply:
Sure, I don't have a good mapping in my head of what syntax is valid where.

I just know that Kurt has frequently corrected by single backticks with double backticks in changelog fragments so I started to default to that in changelogs.